### ISSUE TITLE
Open deep link after click on notification

### ISSUE THREAD

**Issue by jvaclavik** [evollu/react-native-fcm#733]
Is it possible to open the specific deep link after open notification? I didn't find it in docs, but it's better than send some payload and realize redirection on our own in the app according to the payload.

**Comment by evollu** [comment]
yes. you can do navigation in notification callback, make sure wrap the navigation in a timeout

**Comment by zidniryi** [comment]
How bro @evollu  can you give the tutorial or link, thanks, bro.
